File: shoebill_plugins/VehiclePlugin/src/main/java/lt/ltrp/event/VehicleEvent.java
package lt.ltrp.event;


import lt.ltrp.object.LtrpVehicle;

/**
 * @author Bebras
 *         2016.04.19.
 */
public class VehicleEvent extends net.gtaun.shoebill.event.vehicle.VehicleEvent {


    protected VehicleEvent(LtrpVehicle vehicle) {
        super(vehicle);
    }

    @Override
    public LtrpVehicle getVehicle() {
        return (LtrpVehicle)super.getVehicle();
    }
}

File: shoebill_plugins/VehiclePlugin/src/main/java/lt/ltrp/vehicle/event/VehicleEvent.kt
package lt.ltrp.vehicle.event


import lt.ltrp.vehicle.`object`.LtrpVehicle
import net.gtaun.util.event.Event

/**
 * @author Bebras
 * *         2016.04.19.
 * Base event for [LtrpVehicle]
 */
open class VehicleEvent(open val vehicle: LtrpVehicle) : Event() {

}
